optlang/interface.py

```diff
@@ -76,6 +76,56 @@
 }
 
 
+def evaluate_expression(expression):
+    return expression.subs([
+        (sym, sym.value) for sym in expression.free_symbols])
```

Comment:
If the expression contains optlang.Variable's I think this would fail. Maybe use `expression.atoms(SymbolicParameter)` instead